spec/ics-007-tendermint-client/README.md

```diff
@@ -114,14 +124,20 @@ Tendermint client validity checking uses the bisection algorithm described in th
 function checkValidityAndUpdateState(
   clientState: ClientState,
   header: Header) {
-    // assert that header is newer than any we know
-    assert(header.height < clientState.latestHeight)
+    // assert trusting period has not yet passed
+    assert(now - clientState.latestTimestamp < trustingPeriod)
```

Comment:
I'm confused as to what `now` would mean in this context, could you clarify @cwgoes

Reply:
time.Now()

Reply:
The current timestamp. Clarified & added as an explicit requirement to ICS 24.

Reply:
Hmm so `currentTimeStamp` in an SDK chain would be `ctx.BlockHeader().Time()`?

Reply:
Yes, precisely.